stellargraph/losses.py

```diff
@@ -19,21 +19,30 @@
 from .core.experimental import experimental
 
 
-@experimental(reason="lack of unit tests", issues=[804])
-def graph_log_likelihood(y_true, y_pred):
+def graph_log_likelihood(batch_adj, wys_output):
```

Comment:
This is a bit mind-bending to me: the "y_true" are actually just rows from the adjacency matrix, and the "y_pred" are arbitrary scores and the random walk slices, and aren't actually predicting the y_trues at all? Maybe this could be explained a bit more in the doc string?

Reply:
so far I've got: 

```Computes the graph log likelihood loss function as in https://arxiv.org/abs/1710.09599. This is different
    to most keras loss functions in that it doesn't directly compare predicted values to expected values. It uses
    `wys_output` which contains the dot products of embeddings and expected random walks, and part of the adjacency
    matrix `batch_adj` to calculate how well the node embeddings capture the graph structure in some sense.```